src/content/guides/shimming.md

```diff
@@ -13,233 +13,434 @@ related:
+``` diff
+- import 'babel-polyfill';
+-
+  function component() {
+    var element = document.createElement('div');
 
-The simplest way to mitigate this is by adding a separate entry point in your webpack config file including the polyfills your project needs.
+    element.innerHTML = join(['Hello', 'webpack'], ' ');
 
-```javascript
-// webpack.config.js
-module.exports = {
-  entry: {
-    polyfills: [
-      'babel-polyfill',
-      'whatwg-fetch'
-    ],
-    main: './src/index.js'
+    return element;
   }
-  // ... rest of your webpack config
-};
+
+  document.body.appendChild(component());
 ```
 
-An alternative is to create a new entry file and manually import these packages.
+__project__
+
+``` diff
+  webpack-demo
+  |- package.json
+  |- webpack.config.js
+  |- /dist
+  |- /src
+    |- index.js
+    |- globals.js
++   |- polyfills.js
+  |- /node_modules
+```
+
+__src/polyfills.js__
 
 ```javascript
-// src/polyfills.js
 import 'babel-polyfill';
 import 'whatwg-fetch';
 ```
 
-```javascript
-// webpack.config.js
-module.exports = {
-  entry: {
-    polyfills: './src/polyfills.js',
-    main: './src/index.js'
-  }
-  // rest of your webpack config
-};
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+
+  module.exports = {
+-   entry: './src/index.js',
++   entry: {
++     polyfills: './src/polyfills.js',
++     index: './src/index.js'
++   },
+    output: {
+-     filename: 'bundle.js',
++     filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    },
+    module: {
+      rules: [
+        {
+          test: require.resolve('index.js'),
+          use: 'imports-loader?this=>window'
+        },
+        {
+          test: require.resolve('globals.js'),
+          use: 'exports-loader?file,parse=helpers.parse'
+        }
+      ]
+    },
+    plugins: [
+      new webpack.ProvidePlugin({
+        join: ['lodash', 'join']
+      })
+    ]
+  };
 ```
 
-In your html file you need to conditionally load the `polyfills.js` file before your bundle. How you make this decision depends on the technologies and browsers you need to support.
+With that in place, we can add the logic to conditionally load our new `polyfills.bundle.js` file. How you make this decision depends on the technologies and browsers you need to support. We'll just do some simple testing to determine whether our polyfills are needed:
+
+__dist/index.html__
+
+``` diff
+  <html>
+    <head>
+      <title>Getting Started</title>
++     <script>
++       var modernBrowser = (
++         'fetch' in window &&
++         'assign' in Object
++       );
++
++       if ( !modernBrowser ) {
++         var scriptElement = document.createElement('script');
++
++         scriptElement.async = false;
++         scriptElement.src = '/polyfills.bundle.js';
++         document.head.appendChild(scriptElement);
++       }
++     </script>
+    </head>
+    <body>
+      <script src="bundle.js"></script>
```

Comment:
I know that this was already merged, but I just spotted this on the site though I could be wrong... now that the output is `[name].bundle.js` will index bundle be `index.bundle.js` instead of just `index.js`?

Reply:
Yeah, good catch I missed that.